mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -1213,7 +1213,8 @@ def test_add_channels():
     raw_badsf.info['sfreq'] = 3.1415927
     raw_eeg.crop(.5)
 
-    assert_raises(AssertionError, raw_meg.add_channels, [raw_nopre])
+    raw.load_data()
+    assert_raises(RuntimeError, raw_meg.add_channels, [raw_nopre])
```

Comment:
@larsoner can you check if this is as intended?

Reply:
Yes but why do `raw.load_data()` here, it's not used (and already has `load_data()` above anyway)